### ISSUE TITLE
validation performance

### ISSUE THREAD

**Issue by hfutdzx** [Pointcept/PointTransformerV3#77]
excellent work！
i have a question.
How to evaluate the performance of the validation set？
 evaluation is performed on point clouds after grid sampling or Upsampling the predicted results to the original point cloud？

**Comment by Gofinge** [comment]
Yes, an explanation is available here (https://github.com/Pointcept/Pointcept?tab=readme-ov-file#testing)
